src/utils/__tests__/numberify.test.ts

```diff
@@ -6,6 +6,11 @@ describe('numberify()', () => {
     const expected = 123;
     expect(numberify(value)).toEqual(expected);
   });
+  it('should call parseNumber() for numbers with separators and return a number', () => {
+    const value = '1_000_000';
+    const expected = 1000000;
+    expect(numberify(value)).toEqual(expected);
+  });
```

Comment:
Please move this test into `src/utils/parseNumber.ts`.

Reply:
Done ✅